Casks/k/keybase.rb

```diff
@@ -48,10 +48,13 @@
             ]
 
   zap trash: [
+        "~/Library/Application Support/com.apple.sharedfilelist/com.apple.LSSharedFileList.ApplicationRecentDocuments/keybase.*",
```

Comment:
We typically end those entries with `.sfl*`, so this should either be 

```suggestion
        "~/Library/Application Support/com.apple.sharedfilelist/com.apple.LSSharedFileList.ApplicationRecentDocuments/keybase.sfl*",
```

or

```suggestion
        "~/Library/Application Support/com.apple.sharedfilelist/com.apple.LSSharedFileList.ApplicationRecentDocuments/keybase.*.sfl*",
```

depending on whether it has been shortened.